### PR TITLE
Fix sitemap and enable minify

### PR DESCRIPTION
**Pull request by jimmy0251**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func setupRouter(sqlDb *sqlx.DB) *gin.Engine {
 
 	router.POST("/api/send-career-mail", jobsRepo.SendCareerMail)
 
-	router.GET("/sitemap", sitemapRepo.GenerateSitemap)
+	router.GET("/api/sitemap", sitemapRepo.GenerateSitemap)
```
Comment (cp-sumi-k) [inline main.go:53]: We should update API URL in test also

File: nginx/conf.d/default.conf [hunk 2]
```diff
@@ -6,11 +6,6 @@ server {
     location / {
         proxy_pass         http://canopas-website-frontend:3080;
         proxy_set_header   Host $host;
-        proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
```
Comment (cp-sumi-k) [inline nginx/conf.d/default.conf:9]: Why do we need to remove these headers? especially `X-Forwarded-For`? Isn't it will be used for forwarding?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: nginx/conf.d/default.conf (hunk 2 above)
Comment (jimmy0251) [reply nginx/conf.d/default.conf:9]: We don't have any use of these headers. Reverse proxy just works with proxy_pass attribute.

File: main.go (hunk 1 above)
Comment (jimmy0251) [reply main.go:53]: I think we are not testing path in tests. We are testing that path we configure for any endpoint works fine. In short, we are not writing tests for main.go, thus it does not matter much if we update path in tests.

Comment (jimmy0251) [reply main.go:53]: And tests not failing confirms that.

Comment (cp-sumi-k) [reply main.go:53]: Yes, we are not testing the path, but it's for similarity.

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**